website/pages/docs/nia/installation/requirements.mdx

```diff
@@ -40,8 +42,104 @@ The modules listed below are availabe to use and are comptabible with consul-ter
 
 #### Palo Alto Networks
 
-### How to Create a Terraform Module
+### How to Create a Compatible Terraform Module
+
+-> **Note:** Consul-Terraform-Sync is currently in Technology Preview. Specifications in this section may change going into Beta release.
```

Comment:
If we get that section wide tech preview header we should remove this as redundant. But keeping until that is confirmed, it in case we can't for some reason, is a good idea.